astra/astra-inputs/distgen.yaml

```diff
@@ -11,19 +11,31 @@ total_charge:
   units: pC
   value: 100
 
-r_dist:
-  type: radial_uniform
-  max_r:
+xy_dist:
+  file: laser_image.txt
+  type: file2d
+  min_x: 
+    value: -15
     units: mm
-    value: 0.5
-  
+  max_x:
+    value: 15
+    units: mm
+  min_y: 
+    value: -15
+    units: mm
+  max_y:
+    value: 15
+    units: mm
+  threshold: .26
+  #invert: True
+
 t_dist:
   type: gaussian
   avg_t:
     units: ps
     value: 0
   sigma_t:
     units: ps
-    value: 0.85
+    value: 32.87
```

Comment:
Was this the number we calculated? I thought we calculated 32 ps? This should be the 'sigma' value, not the FWHM, so a fraction of 32? It's not a huge problem for your paper/work/test case, but we should fix it before merging.